mosaic/descriptor.py

```diff
@@ -224,11 +246,11 @@ def _compute_cell_patches(ds):
 
 def _compute_edge_patches(ds, latlon=False):
     
-    # account for zeros indexing
-    cellsOnEdge = ds.cellsOnEdge - 1
-    verticesOnEdge = ds.verticesOnEdge - 1
+    # connectivity arrays have already been zero indexed
+    cellsOnEdge = ds.cellsOnEdge
+    verticesOnEdge = ds.verticesOnEdge
     
-    # is this masking sufficent ?
+    # is this masking correct ??
     cellMask = cellsOnEdge <= 0
     vertexMask = verticesOnEdge <= 0
```

Comment:
```suggestion
    cellMask = cellsOnEdge < 0
    vertexMask = verticesOnEdge < 0
```

Reply:
Thanks for catching this!

Making the edge patch construction more robust, for example to better handle coastlines, is definitely on the todo list.